pipeline/snapcraft/snapcraft_test.go

```diff
@@ -17,24 +18,23 @@ func TestDescription(t *testing.T) {
 }
 
 func TestRunPipeMissingInfo(t *testing.T) {
-	for name, snap := range map[string]config.Snapcraft{
-		"missing summary": {
+	for eerr, snap := range map[error]config.Snapcraft{
```

Comment:
There's only one thing I hate about go, and is the short variable names. It's so hard for me to parse them...
I'm +1 on this PR. Thanks!